### ISSUE TITLE
Pulldown not working

### ISSUE THREAD

**Issue by goeldicom** [getgrav/grav-theme-woo#11]
I installed the woo theme 1.3.0 in Grav 1.6.25 and Admin v1.9.14. In the theme configuration I enabled "Dropdown in navbar" and cleared the grav cache.
There is a page called 'docs' (routable, visible, published) with 4 subpages (its parent is 'docs'). In all subpages and also in 'docs' the option 'menu' is enabled and they all are also 'visible'.
The problem is, that in the navbar no dropdown appears.

**Comment by ricardo118** [comment]
yeah i see this template doesnt have dropdown support built in, even tho the setting is there (probably copied from another place when the theme was created)
